packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -19,14 +19,15 @@ import 'ticker_provider.dart';
 ///
 /// The user can transform the child by dragging to pan or pinching to zoom.
 ///
-/// By default, InteractiveViewer may draw outside of its original area of the
+/// By default, InteractiveViewer clip its child using [Clip.hardEdge].
+/// To prevent this behavior, consider setting clipBehavior to [Clip.none],
+/// InteractiveViewer may draw outside of its original area of the
 /// screen, such as when a child is zoomed in and increases in size. However, it
```

Comment:
Now that this is a new sentence (since we changed the previous comma to a period), change this to:

"
When [clipBehavior] is [Clip.none], InteractiveViewer may draw outside of its original area of the screen, such as when a child is zoomed in and increases in size.
"